rest/handler.go

```diff
@@ -189,6 +189,10 @@ func (h *handler) invoke(method handlerMethod) error {
 
 	h.logRequestLine()
 
+	if base.LogKeys["HTTP++"] {
```

Comment:
Oh I just realized that this might be to handle the "don't log HTTP++ with LogStar" scenario.

Looks like we may need another helper method in logger that checks the LogKeys map safely but omits the LogStar check?